veracruz-utils/src/der.rs

```diff
@@ -0,0 +1,151 @@
+//! Some simple manipulations of Distinguished Encoding Rules (DER).
```

Comment:
Before we merge this, can we confirm that MbedTLS doesn't offer these DER-specific functions?

Reply:
> Before we merge this, can we confirm that MbedTLS doesn't offer these DER-specific functions?

It does, but I hadn't merged MbedTLS yet. Now I have, and this PR has become much simpler!